components/user_limit_modal/user_limit_modal.tsx

```diff
@@ -20,7 +22,18 @@ type Props = {
 };
 
 export default function UserLimitModal(props: Props) {
+    useEffect(() => {
+        pageVisited(
+            TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+            'pageview_modal_user_limit_reached',
+        );
+    }, []);
+
     const onSubmit = () => {
+        trackEvent(
+            TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+            'click_modal_user_limit_updrade',
```

Comment:
Still typo on upgrade, could be losing my mind though

Reply:
Removed the g instead of the d. woops.